tools/update_dtb_initrd.py

```diff
@@ -25,10 +25,19 @@ def update_node_property(node, property_name, property_value):
 
 
 def update_chosen(node, initrd_start, initrd_size):
+    # DTS Protocol dictates >32-bit values be represented in pairs. ie, 0x800000000 -> 0x8 0x0
```

Comment:
I wonder if having a generic function would be useful, because other addresses might also have this translation pitfall also. Then a helper function could turn out to be useful:

```
# DTS Protocol dictates >32-bit values be represented in pairs. ie, 0x800000000 -> 0x8 0x0
def address_to_property(address):
    return [address] if address <= 0xFFFFFFFF \
           else [address >> 32, address & 0xFFFFFFFF]

[...]

def update_chosen(node, initrd_start, initrd_size):
    # Update initrd start
    update_node_property(node, "linux,initrd-start", 
                         address_to_property(initrd_start))
    # Update initrd end
    update_node_property(node, "linux,initrd-end",
                         address_to_property(initrd_start + initrd_size))
```

Reply:
Looks like this file is simply a helper file, so the only address range is the initrd. In the future if more functionality is added to the file, then that is a good idea to break this out into a helper.